clang/Util.d

```diff
@@ -80,8 +80,17 @@ mixin template CX ()
 }
 
 extern (C) int mkstemps(char*, int);
+extern (C) char* mkdtemp(char*);
 extern (C) int close(int);
 
+private void randstr (char[] slice)
+{
+    import std.random;
+
+    for (uint i = 0; i < slice.length; ++i)
```

Comment:
I added `randstr` only to fill the range with random characters, however as you showed me a way to do it directly with `uniform`, I removed `randstr` and used uniform directly.